### PR TITLE
!!! Draft Final !!!

### PR DESCRIPTION
**Pull request by ddesmidt**

### CODE REVIEW

File: docs/3.3-Operation-Tools.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+
+## 3.3. Operation Tools
+
+In this section, you'll use 2 popular Operation tools which greatly facilate Network and Security admins:
```
Comment (madhukark) [inline docs/3.3-Operation-Tools.md]: facilitate

File: docs/3.3-Operation-Tools.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+<p align="center">
+  <img width=85% height=85% src="/docs/assets/Graphics/3.3.2.step2.jpg"><br>
+</p>  
+  
+You can also follow each step of the different NSX elements on the bottom half of the screen (and on which device it's running).  
+<p align="center">
+  <img width=85% height=85% src="/docs/assets/Graphics/3.3.2.step3.jpg"><br>
+</p>  
+  
+*Note: The traceflow tracks the different NSX elements up to it reachs the physical fabric, and ends there.*
```
Comment (madhukark) [inline docs/3.3-Operation-Tools.md]: reaches

File: docs/3.2-LogicalNetwork-Security.md [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+</details>
+
+---
+
+### 3.2.4. Configure + Validate Security (Micro-Segmentation)
+
+<details>
+<summary>"Click to expand"</summary>
+
+Follow the procedure detailed in [3.1.3. Configure Microsegmentation](/docs/3.1-Security-Only.md#313-configure-microsegmentation)
```
Comment (madhukark) [inline docs/3.2-LogicalNetwork-Security.md:303]: Micro-Segmentation

**Comment by ddesmidt** [comment]
All good.

**Review by madhukark — APPROVED** [review]

**Pull request merged**